src/freenas/etc/ix.rc.d/ix-autotune

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# $FreeBSD$
+#
+#
+
+# PROVIDE: ix-autotune
+# REQUIRE: NETWORKING
```

Comment:
If we need to reboot we probably want to do it earlier than networking, possibly right after middlewared?